templates/mainapp/layout.html

```diff
@@ -125,7 +125,10 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
                             <p id="bookauthors">
                             </p>
 
-                            <div id="bookrating"></div>
+                            <div class='rating-block-modal' id='rating-block-{{ book.book_id }}' onmouseover="startRatingModal(this)" 
```

Comment:
It's unclear to me how exactly is this `{{ book.book_id }}` being set here, there's no object being iterated for you to access this?
The flow initally to get the book details to modal is like this, whenever the user enters, automatically the corresponding book names link with attributes(see this [line](https://github.com/Praful932/Kitabe/blob/1866e1ff9c8a1164365a449f2c886022a07a65b3/templates/mainapp/layout.html#L247)) are created and when clicked the details are passed onto this function [here](https://github.com/Praful932/Kitabe/blob/1866e1ff9c8a1164365a449f2c886022a07a65b3/templates/mainapp/layout.html#L209), I'm unsure how you are trying to capture the book id here.

Reply:
I forget to delete this, but now it uses `rate-` like classes or id. We use book__id to get id from the book and send an ajax request to API.